### PR TITLE
Completed lc113

### PR DESCRIPTION
**Pull request by kwu25**

### CODE REVIEW

File: kelvinWu/treesDFS/lc113/lc113.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    this.value = value;
+    this.left = null;
+    this.right = null;
+  }
+}
+
+// currentPathArr = [1]    currentSum = 1
+// currentPathArr = [1,7]  currentSum = 8
+// currentPathArr = [1,7,4] currentSum = 12
+
```
Comment (MelissaPastore) [inline kelvinWu/treesDFS/lc113/lc113.js:26]: Nice overview of your approach here.

File: kelvinWu/treesDFS/lc113/lc113.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+var root = new TreeNode(12);
+root.left = new TreeNode(7);
+root.right = new TreeNode(1);
+root.left.left = new TreeNode(4);
+root.right.left = new TreeNode(10);
+root.right.right = new TreeNode(5);
+sum = 23;
+console.log(`Tree paths with sum: ${sum}: ${find_paths(root, sum)}`);
+
```
Comment (MelissaPastore) [inline kelvinWu/treesDFS/lc113/lc113.js:77]: Would be helpful to have a test for your edge case as well.

**Review by MelissaPastore — APPROVED** [review]
